File: app/src/main/java/com/locallinkonline/stcatherineschool/rest/model/DataResourceResponse.java
package com.locallinkonline.stcatherineschool.rest.model;

import com.google.gson.annotations.SerializedName;
import com.locallinkonline.stcatherineschool.room.entity.MenuEntity;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class DataResourceResponse {
    @SerializedName("resourceList")
    List<MenuEntity> resourceList;
}

File: app/src/main/java/com/locallinkonline/stcatherineschool/rest/model/DataResourceResponse.kt
package com.locallinkonline.stcatherineschool.rest.model

import com.google.gson.annotations.SerializedName
import com.locallinkonline.stcatherineschool.room.entity.MenuEntity

data class DataResourceResponse (@SerializedName("resourceList")
                                 var resourceList: List<MenuEntity>?)
